### PR TITLE
Feature: Implementation of LIC4 (#35)

### LINKED ISSUE #35: Implementation of LIC4
Implementation of the class CMW4 is needed.

### PR DESCRIPTION
**Pull request by felixkollin**
Implements LIC4 which tests consecutive points in quadrants. Fixes #35

### CODE REVIEW

File: src/main/java/lic/LIC4.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * @version 1.0
+ */
+public class LIC4 implements LIC {
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean evaluate(Point[] points, Parameters_T parameters) {
+        if(parameters.Q_PTS < 2 || parameters.Q_PTS > points.length) {
```
Comment (phistr2) [inline src/main/java/lic/LIC4.java]: Is it not allowed to have Q_PTS == 1 and QUADS == 0 for example?

File: src/main/java/lic/LIC4.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            return false;
+        }
+        if(parameters.QUADS < 1 || parameters.QUADS > 3) {
+            return false;
+        }
+        if(parameters.Q_PTS <= parameters.QUADS) { // Less points than quadrats we need to exceed
```
Comment (phistr2) [inline src/main/java/lic/LIC4.java]: The wording in the assignement is that we need to have data points that lie in _more_ quads than QUADS, so it should not be possible to have them equal either

File: src/main/java/lic/LIC4.java (hunk 1 above)
Comment (felixkollin) [reply src/main/java/lic/LIC4.java]: Nope, it's stated in the requirement

File: src/main/java/lic/LIC4.java (hunk 2 above)
Comment (felixkollin) [reply src/main/java/lic/LIC4.java]: That is correct, I'll change that

**Pull request merged**